### DISCUSSION TITLE
課題06についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#19] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/commit/76913e2cb45149fdf45720eb7d4737cae5ce273f#commitcomment-77906984

↑のリンク先にて、質問を記載させていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/commit/76913e2cb45149fdf45720eb7d4737cae5ce273f#commitcomment-77933411
